modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -3544,6 +3544,110 @@ RETURN *
 ======
 =====
 
+[#_neo_clientnotification_statement_redundantoptionalprocedure]
+=== Redundant Optional Procedure
+
+.notification details
+[cols="<1s,<4"]
+|===
+|neo4j code
+m|neo.clientnotification.statement.redundantoptionalprocedure
+|title
+a|The use of `OPTIONAL` is redundant when the procedure calls a void procedure.
+|description
+|The use of `OPTIONAL` is redundant as `CALL %s` is a void procedure.
+|category
+m|generic
+|gqlstatus code
+m|03n61
+|status description
+a|info: redundant optional procedure. The use of `OPTIONAL` is redundant as `CALL %s` is a void procedure.
+|classification
+m|generic
+|severitylevel
+m|information
+|===
+
+.Redundant use of optional on a procedure call.
+[.tabbed-example]
+=====
+[.include-with-neo4j-code]
+======
+Query::
++
+[source,cypher]
+----
+OPTIONAL CALL db.createLabel("A")
+----
+
+Description of the returned code::
+Optional is redundant in the case of void procedures. The use of `OPTIONAL` on void procedures have no effect and can be removed.
```

Comment:
Is this description the same as defined in the monorepo? I couldn't find it in the code.